### ISSUE TITLE
Blade Projectile Sprite Bug

### ISSUE THREAD

**Issue by Derpduck** [Gang-Garrison-2/Gang-Garrison-2#220]
The sprite flips left or right depending on which way your character is facing. Was working on a mod and at first thought it was something I had done and forgotten about, I slowed the game down to 6FPS to investigate since it's hard to see at normal speed.

Really this should just be in the create event, but it could be done in a slightly better way anyway rather than checking mouse position.

Though it's possible this is wrong since I've only tested in a mod that changes some things to do with the blade's visuals.

https://github.com/Medo42/Gang-Garrison-2/blob/master/Source/gg2/Objects/Projectiles/BladeB.events/Begin%20Step.xml

**Comment by Medo42** [comment]
No, you're correct. I stumbled over it in the code already, but didn't do anything about it because it didn't seem all that noticable.
